js/src/dom/selectorEngine.js

```diff
@@ -64,21 +120,24 @@ const SelectorEngine = (() => {
         return SelectorEngine.findOne(selector, element)
       }
 
-      return element.querySelectorAll(selector)
+      return findFn.call(element, selector)
     },
 
-    findOne(selector, element = document) {
+    findOne(selector, element = document.documentElement) {
       if (typeof selector !== 'string') {
         return null
       }
 
-      let selectorType = 'querySelector'
```

Comment:
Not a big fan of that, because if it's an ID you'll use `querySelectorAll` instead of `getElementById` which is faster

Reply:
Multiple selector starting with `#` (ex: `#id > li, #id1 > .active`) will match and return a wrong result

Reply:
However, I think this is in the wrong PR, i've encountered an error on this while working on ScrollSpy 😅

Reply:
@Johann-S Should I revert this? Or fix testing the selector against a regex that excludes complex selectors?

Reply:
Nope finally I think it's fine like this 👍